lib/generator/command_line/generator_optparser.rb

```diff
@@ -33,10 +33,6 @@ def parse_options
     def option_parser
       @option_parser ||= OptionParser.new do |parser|
         parser.banner = "Usage: #{$PROGRAM_NAME} [options] exercise-generator"
-        parser.on('-u', '--update', 'Update test version') { |value| options[:update] = value }
-        parser.on('-a', '--all', 'Regenerate all available test suites (does not update version)') do |value|
```

Comment:
Oops, I guess I was not clear enough here: `--all` still needs to be an option, but the "does not update version" message can go.

Reply:
I suggest reverting https://github.com/exercism/ruby/pull/877/commits/3e14ddf1611953108c569f44be2327dd077a4247 and trying again to just remove the `-u` options.

Reply:
Hi there! sorry for my delay, sorry about that I'll update tonight!